fews_3di/tests/example_settings.xml

```diff
@@ -2,8 +2,7 @@
     <ns0:startDateTime date="2020-01-26" time="10:00:00"/>
     <ns0:endDateTime date="2020-01-30" time="12:00:00"/>
     <ns0:properties>
-        <ns0:string key="username" value="pietje" />
-        <ns0:string key="password" value="onder-de-deurmat" />
+        <ns0:string key="api_token" value="token" />
```

Comment:
Dit is prima, maar ik zou de oorspronkelijke example_settings.xml, dus met user/pass, als `settings_with_user_pass.xml` bewaren en dan een test toevoegen dat die een error oplevert omdat er een gebruikersnaam in staat. Dan kan je "je gebruikt nog user/pass ipv apikey" zeggen i.p.v. alleen "api key mist".

=> extra settings file + code voor de username-detectie + error + test.

Reply:
Reden: goede plek om de overgang naar het nieuwe systeem mogelijk te maken.
Maar... misschien is het teveel moeite. Als het makkelijk kan zou ik het doen.

Reply:
Ik heb zoiets toegepast en een testje toegevoegd in de test_utils.